QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+        private const string QuestionCommand = "/question";
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public string ReceiveMessage(string message){
+            string outputMessage;
+
+            if (String.IsNullOrEmpty(message) || !message.StartsWith(QuestionCommand)){
+                return null;
+            }
+
+            string questionText = message.Remove(0, QuestionCommand.Length);
+            questionText = questionText.TrimStart(' ','\u0009');
+            questionText = questionText.TrimEnd(' ', '\u0009');
```

Comment:
What about newline characters? :)

The trick here is that you can't (well, shouldn't) list all possible whitespace characters we could encounter. Luckily there is a built in method, [`Trim`](https://msdn.microsoft.com/en-us/library/t97s7bs3(v=vs.110).aspx) that does exactly what we want. From the docs:

> The Trim method removes from the current string all leading and trailing white-space characters.

Since this is a pretty common operation, most languages will have a built-in trim method to handle this for us.